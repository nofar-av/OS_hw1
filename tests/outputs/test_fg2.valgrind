==26960== Memcheck, a memory error detector
==26960== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26960== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26960== Command: /home/omer/OS_wet1/smash
==26960== Parent PID: 26956
==26960== 
==26960== 
==26960== FILE DESCRIPTORS: 4 open at exit.
==26960== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_fg2.valgrind
==26960==    <inherited from parent>
==26960== 
==26960== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_fg2.err
==26960==    <inherited from parent>
==26960== 
==26960== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_fg2.out
==26960==    <inherited from parent>
==26960== 
==26960== Open file descriptor 0:
==26960==    <inherited from parent>
==26960== 
==26960== 
==26960== HEAP SUMMARY:
==26960==     in use at exit: 0 bytes in 0 blocks
==26960==   total heap usage: 76 allocs, 76 frees, 84,788 bytes allocated
==26960== 
==26960== All heap blocks were freed -- no leaks are possible
==26960== 
==26960== For counts of detected and suppressed errors, rerun with: -v
==26960== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
