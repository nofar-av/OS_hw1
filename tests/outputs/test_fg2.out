smash> smash> smash> smash> smash> smash> sleep 300& : 5469
smash: got ctrl-C
smash: process 5469 was killed
smash> sleep 100& : 5466
smash: got ctrl-C
smash: process 5466 was killed
smash> sleep 200& : 5468
smash: got ctrl-C
smash: process 5468 was killed
smash> sleep 400& : 5470
smash: got ctrl-C
smash: process 5470 was killed
smash> sleep 500& : 5472
smash: got ctrl-C
smash: process 5472 was killed
smash> smash: sending SIGKILL signal to 0 jobs:
